gammapy/modeling/models/temporal.py

```diff
@@ -174,10 +175,12 @@ def integral(self, t_min, t_max, oversampling_factor=100, **kwargs):
         norm : float
             Integrated flux norm on the given time intervals
         """
-        t_values = np.linspace(t_min.mjd, t_max.mjd, oversampling_factor, axis=-1)
+        t_values, steps = np.linspace(
+            t_min.mjd, t_max.mjd, oversampling_factor, retstep=True, axis=-1
+        )
         times = Time(t_values, format="mjd")
         values = self(times)
-        integral = np.sum(values / oversampling_factor, axis=-1)
+        integral = np.sum(values, axis=-1) * steps
```

Comment:
This change seems to cause the changes in the test values. But looking at the implementation, it seems, that now it is correct. Previously it was `1 / factor`, so divided by the number of nodes (including bounds), while now it is `1 / (factor - 1)`, which the number of bins themselves. So looks good to me!